### PR TITLE
fix: CFG-28 Uncaught TypeError

### PR DESCRIPTION
**Pull request by zoeradkani**
# Problem
Customer support ticket for Uncaught TypeError on Country Selector, when no Result is returned.
[Issue Here](https://github.com/onfido/onfido-sdk-ui/issues/1659)
# Solution
trying to catch the undefined property.
## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by DannyvanderJagt** [comment]
Any idea about the underlaying issue causing this?

**Comment by zoeradkani** [comment]
> 

@DannyvanderJagt I could not reproduce it this time around, which I think it was due to having a new build and updated packages, I certainly had seen it before. From what I remember the translate was returning `undefined`  on `tNoResults` when the whole component was not rendered fully.
I suppose the simplest other solution is just return the a translate on the `AutoComeplete` component instead if calling getNoResultsTextForDropdown(), something like `tNoResults={() => translate('country_select.alert_dropdown.country_not_found')}`, what do you think?
CC: @nulrich

**Comment by DannyvanderJagt** [comment]
Ah okay, I tested a couple of versions of 6.15.5 and fiddled with the source code of the autocomplete package but couldn't recreate the error either.

File: src/components/CountrySelector/index.tsx [hunk 1]
```diff
@@ -126,11 +126,15 @@ class CountrySelection extends Component<Props, State> {
     return hasOnePreselectedDocument(steps) && documentType !== 'passport'
   }
 
-  getNoResultsTextForDropdown = () =>
-    parseTags(
+  getNoResultsTextForDropdown = () => {
+    if (typeof this.props.translate === undefined) {
```
[the remark this answers is not in the document]
Comment (nulrich) [reply src/components/CountrySelector/index.tsx:130]: Ok, we'll remove it when we'll move to translate Context

**Review by nulrich — APPROVED** [review]

**Pull request merged**